### PR TITLE
Patch Env CLI

### PR DESCRIPTION
**Pull request by RMartinOscar**
- [x] Remove `--use-redis` following [this commit](https://github.com/pelican-dev/panel/commit/953ee940aa5955dfdad452dbdc46f48ca5e5bb55#diff-ee666c86898d686c606a20796d57bbb363ead4dd6dfa14bbd1a6764d513b5edfL17)
- [x] Add Redis Username (default if empty)
- [x] Move confirm prompt to lang for reuse

### CODE REVIEW

File: app/Traits/Commands/RequestRedisSettingsTrait.php [hunk 1]
```diff
@@ -12,19 +12,34 @@ protected function requestRedisSettings(): void
...
+        }
         if ($askForRedisPassword) {
             $this->output->comment(__('commands.appsettings.redis.comment'));
             $this->variables['REDIS_PASSWORD'] = $this->option('redis-pass') ?? $this->output->askHidden(
                 'Redis Password'
             );
         }
 
+        if (empty($this->variables['REDIS_USERNAME'])) {
+            $this->variables['REDIS_USERNAME'] = 'default';
```
Comment (Boy132) [inline app/Traits/Commands/RequestRedisSettingsTrait.php:41]:
```suggestion
            $this->variables['REDIS_USERNAME'] = 'null';
```

**Review by Boy132 — CHANGES_REQUESTED** [review]
Looks good, but the default value for the redis user should be empty/ null.

File: app/Traits/Commands/RequestRedisSettingsTrait.php (hunk 1 above)
Comment (RMartinOscar) [reply app/Traits/Commands/RequestRedisSettingsTrait.php:41]: Tried that and it didn't work i had to set it to 'default'

**Review by Boy132 — APPROVED** [review]
LGTM

**Pull request merged**